manifests/redhat.pp

```diff
@@ -15,15 +15,15 @@
     case $agent_major_version {
       5 : {
         $defaulturl = "https://yum.datadoghq.com/rpm/${::architecture}/"
-        $gpgkey = 'https://yum.datadoghq.com/DATADOG_RPM_KEY.public'
+        $gpgkeys = ['https://yum.datadoghq.com/DATADOG_RPM_KEY.public']
```

Comment:
If we can specify multiple keys in the repo file, let's list all the keys we want trusted (same as what we do in the [install script](https://github.com/DataDog/datadog-agent/pull/6497).

For Agent 5 and Agent 6, that means all three keys; for Agent 7, the latest two keys.

Reply:
We can (with the same kind of hack you use on DataDog/datadog-agent#6497), you just join the keys when passing them to the repo object:
https://github.com/DataDog/puppet-datadog-agent/blob/5853bf4cacd17454f19b3aee51832527f0d6d8ae/manifests/redhat.pp#L69

I am going to try to

- Add the keys on each list
- Remove the manual `rpm import`

and then install Agent 5, 6 and 7 to see if it can work that way

Reply:
Done